monster-cards/src/main.ts

```diff
@@ -2,38 +2,94 @@ import createApp from 'dojo-app/createApp';
 
 import router from './routes';
 import cardStore, { bindActions as bindCardStoreActions } from './stores/cardStore';
+import favStore, { bindActions as bindFavStoreActions } from './stores/favoriteStore';
+import createVNodeEvented from 'dojo-widgets/mixins/createVNodeEvented';
+import createCssTransitionMixin from 'dojo-widgets/mixins/createCssTransitionMixin';
 
-import createNavbar from './widgets/navbar/createNavbar';
 import createCardDescription from './widgets/card-details/createCardDescription';
 import createContainer from './widgets/common/createContainer';
+import createSearchInput from './widgets/common/createSearchInput';
+import createIcon from './widgets/common/createIcon';
+import createNavMenu from './widgets/navbar/createNavMenu';
 import createCard from './widgets/card/createCard';
 import createCardSummary from './widgets/card/createCardSummary';
+import createCardFavorite from './widgets/card/createCardFavorite';
 import defaultWidgetStore from './stores/widgetStore';
+import { showFavorites, hideFavorites } from './actions/userActions';
 
 import 'maquette/src/css-transitions';
 
 const app = createApp({ defaultWidgetStore });
 
 app.registerStore('cards-store', cardStore);
+app.registerStore('fav-store', favStore);
 app.loadDefinition({
 	customElements: [
 		{
 			name: 'milestone-card',
 			factory: createCard
 		},
 		{
+			name: 'fav-card',
+			factory: createCardFavorite
+		},
+		{
 			name: 'card-description',
 			factory: createCardDescription
 		},
 		{
 			name: 'milestone-card-summary',
 			factory: createCardSummary
+		},
+		{
+			name: 'fav-milestone-card',
+			factory: createCardSummary
 		}
 	],
 	widgets: [
 		{
 			id: 'navbar',
-			factory: createNavbar
+			factory: createContainer,
+			options: {
+				tagName: 'header'
+			}
+		},
+		{
+			id: 'navbar-menu',
+			factory: createNavMenu
+		},
+		{
+			id: 'navbar-actions',
+			factory: createContainer,
+			options: {
+				tagName: 'ul'
+			}
+		},
+		{
+			id: 'navbar-search',
+			factory: createSearchInput
+		},
+		{
+			id: 'navbar-favorites',
+			factory: createContainer.mixin(createVNodeEvented),
+			options: {
+				tagName: 'li'
+			},
+			listeners: {
+				mouseover: () => { showFavorites.do(); },
```

Comment:
That's not quite right, but it can be `mouseover: showFavorites` (minus the `.do`)